hoodie-client/src/main/java/com/uber/hoodie/table/HoodieTable.java

```diff
@@ -120,6 +124,17 @@ public TableFileSystemView getFileSystemView() {
     return new HoodieTableFileSystemView(metaClient, getCompletedCommitTimeline());
   }
 
+  /**
+   * Get the read optimized view of the file system for this table
+   */
+  public SealedLatestVersionOnlyFSView getLatestFileSliceOnlyFSView() {
+    if (null == latestFileSliceOnlyFSView) {
```

Comment:
just a thought.. do we need to protect this synchronized? probably not, since driver access would be single threaded anyway.